drivers/wifi/nxp/nxp_wifi_drv.c

```diff
@@ -1772,6 +1779,7 @@ static const struct zep_wpa_supp_dev_ops nxp_wifi_drv_ops = {
 	.dpp_listen               = wifi_nxp_wpa_dpp_listen,
 	.remain_on_channel        = wifi_nxp_wpa_supp_remain_on_channel,
 	.cancel_remain_on_channel = wifi_nxp_wpa_supp_cancel_remain_on_channel,
+	.send_action_cancel_wait  = wifi_nxp_wpa_supp_cancel_action_wait,
```

Comment:
Where is the implementation of this function? I don't see it within Zephyr or NXP's HAL

Reply:
https://github.com/zephyrproject-rtos/hal_nxp/pull/416
it's in this PR.

Reply:
Ok, so we need to make this PR dependent on https://github.com/zephyrproject-rtos/hal_nxp/pull/416, then- otherwise this change is effectively adding dead code that can't be built (unless I'm missing something?)